circuitpython/CIRC02/8_leds.py

```diff
@@ -63,28 +57,31 @@ def oneAfterAnotherNoLoop():
     ledpins[0].value = False
     time.sleep(delayTime)
 
-# turns on one LED after another with a loop
+
 def oneAfterAnotherLoop():
+    """turns one LED on at a time, looping at the end."""
     delayTime = 0.1
-    # iterates through ledpins list
     for led in ledpins:
         led.value = True
         time.sleep(delayTime)
-    # iterates through ledpins list backwards
     for led in ledpins[::-1]:
         led.value = False
         time.sleep(delayTime)
 
+
 def oneOnAtATime():
+    """turns one LED on at a time, looping at the end."""
     delayTime = 0.1
```

Comment:
should we be using pythonic variable name delay_time instead of delayTime?

Reply:
Variables should be lower snake case, so we should be using: `delay_time`